rs/backend/src/accounts_store.rs

```
@@ -1513,6 +1519,14 @@ impl AccountsStore {
             _ => {}
         };
     }
+    fn assert_pre_migration_limit(&self) {
+        let db_accounts_len = self.accounts_db.db_accounts_len();
+        assert!(
+            db_accounts_len < PRE_MIGRATION_LIMIT,
+            "Pre migration account limit exceeded {}",
```

Shouldn't we return a distinctive error message that way the frontend can interpret it easily and display gracefully an explanation of the error to the users?

---

Side note: I'm guessing we `trap` because we do not want to change the interface and return a `Result`, correct?

---

Potentially, yes, but we have time to implement that.